cmd/mcp-server/main.go

```diff
@@ -47,6 +48,9 @@ var (
 			}
 		},
 	}
+
+	allowedTags   = []string{"AI"}
+	swaggerDocURL = "https://api2.edgedelta.com/swagger/doc.json"
```

Comment:
why this url is api2 but not api? If api does not support it, do we know why?

Reply:
It lefu from swagger_internal. But, api is working so let me change it as api.